src/renderer/components/+preferences/preferences-navigation/extensions-preferences-navigation-item.injectable.ts

```diff
@@ -25,7 +26,7 @@ const extensionPreferencesNavigationItemRegistratorInjectable = getInjectable({
           label: `${extension.name}`,
           navigate: () => navigateToExtensionPreferences(extension.sanitizedExtensionId),
           isActive: computed(() => false),
-          isVisible: computed(() => true),
+          isVisible: computed(() => isVisible),
```

Comment:
It would be nice to see a unit test within this commit to see that this change is tested, and also study the test to further understand the motivation behind this change.